lua/nvim_lsp/rust_analyzer.lua

```diff
@@ -1,11 +1,86 @@
 local configs = require 'nvim_lsp/configs'
 local util = require 'nvim_lsp/util'
+local server_name = 'rust_analyzer'
+local release_url = 'https://api.github.com/repos/rust-analyzer/rust-analyzer/releases/latest'
+local postfix = '-linux'
+local bin = 'rust-analyzer'
+
+if vim.fn.has('mac') then
+  postfix = '-mac'
+elseif vim.fn.has('win32') or vim.fn.has('win64') then
+  postfix = '-windows.exe'
+end
```

Comment:
I was wrong about comparing to `1` being redundant, sorry...

Reply:
No problem :+1: